src/automation.md

```diff
@@ -0,0 +1,107 @@
+  - A workflow that uses actions
+
+A workflow can have as many actions as needed and each action has it's own purpose
+
+### Types of Actions
+
+Actions come in two flavours: **"Container Actions and Javascript Actions"**
+
+#### Container Actions
+
+Docker Container actions have the advantage to be packaged with the GitHub Actions code and can only execute in the **Github-Hosted Linux Environment**. The development of this type of action requires a previous understanding of how Docker works. For more information click [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-docker-container-action)
+
+#### Javascript Actions
+
+It is the same as above, but with the environment decoupled from Github allowing faster execution but also accepting great dependency management responsability. For this onboarding document, let's focus on this type of Action, and straight to work!
+
+## Start your workflow file
+
+On your repository create a ```.github/workflows/main.yml``` file and add the following code inside:
+
+```sh
+name: My Simple Workflow
+on: push
+```
+
+Feeling lost ? Don't worry, I will tell you what you did!
+
+```name``` gives you the workflow name. This name will appear on each and every Pull Request or in your Actions Tab. Rename it whatever you want! It's really important when you have multiple workflows on your repository. <br/>
+```on: push``` means that your workflow will only execute when code is pushed in your repository. i.e, using the ```push``` event.
+
+It doesn't seem much, right ? So let's add some more jobs to this workflow! 💼
+
+Inside your ```./github/workflows/main.yml``` append:
+
+```sh
+jobs:
+  build:
+    name: My Simple Workflow
+    runs-on: ubuntu-latest    
+    steps:
+    - uses: actions/checkout@v1
+      env:
+        MY_NAME: "Skilled Gorilla"
+```
+
+Nice job so far! Let's get into more detail on what you just did:
+
+```job``` is the base component of a workflow run <br />
+```build``` is the indentifier to where you're attaching this job<br />
+```name``` is the name of the job which will be displayed in your Actions tab when the workflow is running<br />
+```steps``` the sequence of operations that make up a job<br />
+```uses: actions/checkout@v1``` reference a community action called checkout and it's content's repository<br />
+```env``` is used to specify the environment variables that will be available to your action in runtime<br />
+
+## Keeping a secret 🗝️
+
+The internet is a volatile thing, and sometimes you just need to keep information to yourself. If you don't feel like sharing your tokens, passwords or sensible information don't worry, I got you covered!<br/>
+
+On your project's navigation tab select **Settings** and them, on your left sidebar click on **Secrets**.
+The **Secrets** tab will show you a simple key/value pair where you can safely store all sensible information relevant to your project. To reference that on your workflow simply add:
```

Comment:
> To reference that

Should be:
To reference these

Reply:
I would personally do something like:

> ...simply add a `TOKEN` value, like so:

This way we are clearly pointing the user to the portion that matters to them.